src/system/ExtensionsModule/Schema/schema.composer.json

```diff
@@ -73,6 +73,7 @@
                         "class": {"type": "string"},
                         "displayname": {"type": "string"},
                         "url": {"type": "string"},
+                        "icon": {"type": "string"},
```

Comment:
`"icon"` should be added to the `"required"` list on line 95.

now, I'm not sure about my comment above about `url`. If it is not used and not required, it should be removed here too (line 75)... and then checked in other extensions... but not sure

Reply:
The `required` section is only for 1st level (see it contains `extra` for example), but not for sub elements.

Reply:
sorry - I meant line 87

Reply:
added in 58aabf244ba8fdb03b092e52fb8826eec178a2cf